### DISCUSSION TITLE
Translations reviewer

### DISCUSSION THREAD

**Discussion by ghost** [hluk/CopyQ#1767] [General]
Hello @hluk ,

I'm using CopyQ in Portuguese (Portugal), but I find lots of typos. Could you make me a reviewer on Weblate, so that someone (me) would maintain the translation clean?

Thank you,
Pedro Albuquerque (Weblate user @PedroAlbuquerque)

**Comment by hluk** [comment]
Not sure what is the problem exactly. Can you not update the translation on Weblate?

**Reply by ghost** [reply]
Yes, but so can anyone else, which is why there are so many typos.

**Reply by hluk** [reply]
OK, let's try it. What is your username on Weblate?

**Reply by ghost** [reply]
Its on the first post, @PedroAlbuquerque

**Reply by hluk** [reply]
Done. I've added you as reviewer.

**Reply by ghost** [reply]
Thank you.
Fully translated and reviewed. It's ready to use.

**Reply by Pedroalbuquerque** [reply]
I guess this thread is misplaced... I am PedroAlbuquerque, but I do not understand any of this discussion :) 
I guess there are more than one PedroAlbuquerque.

**Comment by Pedroalbuquerque** [comment]
Sorry, my mistake, the thread might be ok, but I am getting notification maybe due to @Pedroalbuquerque  reference.
